Gemfile.tt

```diff
@@ -5,12 +5,12 @@ ruby "<%= RUBY_VERSION %>"
 
 # Core
 gem 'bootsnap', ">= 1.7.5", require: false
-gem 'pg'<%= gemfile_requirement("pg") %>
-gem 'puma'<%= gemfile_requirement("puma") %>
+gem 'pg', '~> 1.3.4'
+gem 'puma', '~> 5.6'
```

Comment:
What is the reasoning for removing the helper method `gemfile_requirement`?

Reply:
The reason was that it wasn't adding the latest gem version. I think we either need to rewrite that function, or update the versions manually, when updating the template.

Reply:
I think you might have misunderstood what `gemfile_requirement` does and how the version requirements notation of bundler works.

When testing locally with `gemfile_requirement` in place, I get the latest versions of pg and puma installed. As required by default for Rails 7. 
Although in the Gemfile pg gem is specified like `gem "pg", "~> 1.1"` and the latest version is 1.3.5 that doesn't mean that `pg v1.1.x` is installed. The gem notation `~> 1.1` means that the highest available version of 1.x.x will be installed but not above 1, so if pg would be released as a 2.0.0 and the last 1 version is 1.90.12 then this version requirement won't go to version 2.0.0.

So in order to make the template more flexible for future rails versions, I suggest we keep `gemfile_requirement` as it is.